test/upgradability/upgradability.proxy.ts

```diff
@@ -308,12 +309,72 @@ contract("SavingAccount() proxy", async (accounts) => {
             });
 
             // call initFINAddress() & verify FIN address after upgrade
-            await SAV.initFINAddress();
+            await SAV.initFINnCOMPAddresses();
             const FINAddrAfter = await SAV.FIN_ADDR();
             expect(FINAddrAfter).to.be.equal(FINAddress);
 
             // call initFINAddress() again
-            await expectRevert(SAV.initFINAddress(), "Already init");
+            await expectRevert(SAV.initFINnCOMPAddresses(), "Already init");
+        });
+
+        it("SavingAccount - initCOMPAddress(): from V1.1 to latest", async () => {
```

Comment:
It seems there are two tests for single function `initFINnCOMPAddresses`. I think we would just need single test in which it will validate both the addresses.

Reply:
sure, I'll make that change